src/main/java/com/openshift/internal/restclient/okhttp/ResponseCodeInterceptor.java

```diff
@@ -56,7 +61,9 @@ public Response intercept(Chain chain) throws IOException {
 				response = makeSuccessIfAuthorized(response);
 				break;
 			default:
-				throw createOpenShiftException(client, response, null);
+				if ( response.request().tag() instanceof Ignore == false ) {
```

Comment:
Hmm... A not so obvious way to pass around a context.  Okhttp api docs could be better... WFM